libkarel/libkarel_test.py

```diff
@@ -53,6 +53,35 @@ def test_basic(self):
         self.assertEqual(karel_in.zumbadores(1, 1), 0)
         self.assertEqual(karel_in.dump(1, 1), False)
 
+    def test_walls(self):
+        """Prueba de las paredes."""
+
+        karel_in = libkarel.KarelInput('''
+<ejecucion>
+  <condiciones instruccionesMaximasAEjecutar="10000000" longitudStack="65000"></condiciones>
+  <mundos>
+    <mundo nombre="mundo_0" ancho="3" alto="2">
+      <pared x1="1" y1="0" y2="1"></pared>
+      <pared x1="1" y1="1" x2="2"></pared>
+      <pared x1="2" y1="1" y2="2"></pared>
+    </mundo>
+  </mundos>
+  <programas tipoEjecucion="CONTINUA" intruccionesCambioContexto="1" milisegundosParaPasoAutomatico="0">
+    <programa nombre="p1" ruta="{$2$}" mundoDeEjecucion="mundo_0" xKarel="1" yKarel="1" direccionKarel="NORTE" mochilaKarel="0"></programa>
+  </programas>
+</ejecucion>
+        ''')
+
+        Direccion = libkarel.Direccion  # pylint: disable=C0103
```

Comment:
esto es equivalente a

```python
from libkarel import Direccion
```

y así te evitas deshabilitar el linter. es más idiomático además!

Reply:
y libkarel sigue in scope? no hice eso porque no sabía si con eso no iban a poder llamar a las demás cosas de ese módulo

Reply:
sí! :D

Reply:
me imagino que la solución es dejar tanto `import libkarel` como `from libkarel import Direccion`?